config/html.xsl

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0'?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
+
+<xsl:import href="/usr/share/xml/docbook/stylesheet/nwalsh/html/docbook.xsl"/>
```

Comment:
Let's use chunked html where every section is a separate file, example is [here](https://github.com/worldmind/ratiocracy.book/blob/main/xsl/html-params.xsl)